From c72ceb8e9536cdb659021e6d27665876377357ed Mon Sep 17 00:00:00 2001
From: Andre McCurdy <armccurdy@gmail.com>
Date: Mon, 4 Apr 2016 18:01:04 -0700
Subject: [PATCH] v3d/platform/nexus/platform_nexus.mk: dont strip libnxpl.so

Stripping libraries should happen during packaging, or as part of a
post-processing step when creating the rootfs, not when the binaries
are built (even for release builds).

Fix for QA warnings such as:

  WARNING: broadcom-refsw-15.2-r0 do_package: QA Issue: File '/usr/lib/libnxpl.so' from broadcom-refsw was already stripped, this will prevent future debugging! [already-stripped]

Signed-off-by: Andre McCurdy <armccurdy@gmail.com>
---
 rockford/middleware/v3d/platform/nexus/platform_nexus.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rockford/middleware/v3d/platform/nexus/platform_nexus.mk b/rockford/middleware/v3d/platform/nexus/platform_nexus.mk
index 724da46..b939bc7 100644
--- a/rockford/middleware/v3d/platform/nexus/platform_nexus.mk
+++ b/rockford/middleware/v3d/platform/nexus/platform_nexus.mk
@@ -46,7 +46,7 @@ OBJDIR ?= obj_$(NEXUS_PLATFORM)_debug
 LIBDIR ?= lib_$(NEXUS_PLATFORM)_debug
 else
 CFLAGS += -g -Os -DNDEBUG
-LDFLAGS += -g -s
+LDFLAGS += -g
 OBJDIR ?= obj_$(NEXUS_PLATFORM)_release
 LIBDIR ?= lib_$(NEXUS_PLATFORM)_release
 endif
-- 
1.9.1

